spec/support/examples_group/shared_examples_for_ownership_mixin.rb

```diff
@@ -40,6 +40,11 @@
           expect(described_class.user_or_group_owned(user, nil)).to eq([user_owned])
         end
 
+        it "with mixed case userid" do
+          user.update(:userid => "MixedCase")
```

Comment:
this test would pass even without the changes.

Is there a way to test that this change makes something work?